### PR TITLE
Adding a contribution

### PR DESCRIPTION
**Pull request by btb331**
Adding my own version,  They evolve size which speed is derived from.  The faster they are, they faster they lose health.
Predators are white and evolve the best prey to choose.  Their cloning rate is dependant on their health, so if there is lots of prey they multiply quickly.
 
Note: This is my first foray with GitHub so apologies if I've done it wrong

### CODE REVIEW

File: _CodingChallenges/69.4-steering-evolution.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ parts:
...
   url: "/CodingChallenges/69.5-steering-evolution"
+  
+  contributions:
+  - title: "My Evolutionary Steering Agents with predators"
+    author:
+      name: "Ben Ashworth"
+    url: "https://talkingchef.co.uk/codingTrain/genEco/"
+    
+    
+---
```
Comment (nilsweb) [inline _CodingChallenges/69.4-steering-evolution.md]: This line is duplicated, please remove it.

File: _CodingChallenges/69.4-steering-evolution.md [hunk 2]
```diff
@@ -31,6 +31,15 @@ parts:
   url: "/CodingChallenges/69.3-steering-evolution"
 - title: "Evolutionary Steering Behaviors - Part 5"
   url: "/CodingChallenges/69.5-steering-evolution"
+  
+  contributions:
+  - title: "My Evolutionary Steering Agents with predators"
+    author:
+      name: "Ben Ashworth"
+    url: "https://talkingchef.co.uk/codingTrain/genEco/"
+
```
Comment (nilsweb) [inline _CodingChallenges/69.4-steering-evolution.md]: If you have a link to the source code of your contribution, you can provide it with `source: your-url-here`.

File: _CodingChallenges/69.4-steering-evolution.md (hunk 1 above)
Comment (btb331) [reply _CodingChallenges/69.4-steering-evolution.md]: This is done now

File: _CodingChallenges/69.4-steering-evolution.md (hunk 2 above)
Comment (btb331) [reply _CodingChallenges/69.4-steering-evolution.md]: I'm afraid i don't

**Comment by btb331** [comment]
It's now saying there is a conflict. Shall I resolve it, or does admin do that?

**Comment by shiffman** [comment]
@btb331 go ahead and resolve!

**Comment by btb331** [comment]
Thanks Daniel! I've resolved the conflict now so should be all good. :)

**Comment by shiffman** [comment]
Looks great, thank you!

**Comment by btb331** [comment]
Did you look at my example? Did you like it?

**Comment by shiffman** [comment]
yes, it's amazing! Love all the stats.

**Pull request merged**